src/lib/components/DashNgl.react.js

```diff
@@ -0,0 +1,253 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Stage, Selection} from 'ngl';
+
+/**
+ * Dash ngl component
+ */
+export default class DashNgl extends Component {
+    // constructor might not be needed anylonger:
+    // https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599
+    constructor(props) {
+        super(props);
+        this.state = {stage: null, structuresList: []};
+        console.log(this.props);
```

Comment:
Please remove these calls to `console.log`! There are quite a few other ones in the file.